hummingbot/market/dolomite/dolomite_market

```diff
@@ -0,0 +1,1362 @@
+                self.logger().network(
+                    f"Error fetching status update for the order {tracked_order.client_order_id}: "
+                    f"{order_update}.",
+                    app_warning_msg=f"Failed to fetch status update for the order {tracked_order.client_order_id}. "
+                                    f"Check Ethereum wallet and network connection."
+                )
+                continue
+
+            # Check the exchange order ID against the expected value.
+            exchange_order_id = order_update["data"]["order_hash"]
+            if exchange_order_id != tracked_order.exchange_order_id:
+                self.logger().network(f"Incorrect exchange order id '{exchange_order_id}' returned from get order "
+                                      f"request for '{tracked_order.exchange_order_id}'. Ignoring.")
+                
+                
+
+                # Capture the incorrect request / response conversation for submitting to Dolomite.
+                
+                request_url = "%s/v1/orders/addresses/" + Web3Wallet.address % (self.DOLOMITE_REST_ENDPOINT,)
+                
+                response = self._api_response_records.get(request_url)
+                
+                found = False
+                
+                for order in response:
+                    if order["data"]["order_hash"] == tracked_order.exchange_order_id:
+                        found = True
+                        self.logger().network(f"Captured erroneous order update request/response. "
+                                              f"Request URL={response.real_url}, "
+                                              f"Request headers={response.request_info.headers}, "
+                                              f"Response headers={response.headers}, "
+                                              f"Response data={repr(response._body)}, "
+                                              f"Decoded order update={order_update}.")
+                if found == False:
+                    self.logger().network(f"Failed to capture the erroneous request/response for getting the order update "
+                                          f"of the order {tracked_order.exchange_order_id}.")
+
+                continue
+                
+                
+                
+
+            # Calculate the newly executed amount for this update.
+            previous_is_done = tracked_order.is_done
+            
+            
+            #Not blockchain confirmed!
+            new_filled_amount = float(order_update["data"]["dealt_amount_primary"]["amount"]) / math.pow(10, 18)
+            
+            
+            #Equals how much more of (primary ticker) amount has been filled 
+            execute_amount_diff = new_filled_amount - float(tracked_order.executed_amount)
+
+
+            
+            
+            execute_price = 0.0 #Placeholder for market orders (very high / low)
```

Comment:
You can use `market_order_effective_price` for getting the `executed` price for a `MARKET_ORDER`